node/Cargo.toml

```
@@ -78,3 +78,13 @@ rev = "dc4a8329b9c47790c25429a5df272b3648275c1b"
 git = "https://github.com/paritytech/substrate"
 rev = "dc4a8329b9c47790c25429a5df272b3648275c1b"
 package = "substrate-transaction-pool"
+
+[dependencies.grandpa]
```

I’d like to avoid package name aliases because they are very confusing. Same applies to the other packages.
```suggestion
[dependencies.substrate-finality-grandpa]
```